CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 - [csi-driver #809](https://github.com/kubernetes-sigs/blob-csi-driver/issues/809) Fail to mount when uid/gid are provided on command line.
 - [#1032](https://github.com/Azure/azure-storage-fuse/issues/1032) `mount all` CLI params parsing fix when no `config-file` and `tmp-path` is provided.
 - [#1015](https://github.com/Azure/azure-storage-fuse/issues/1015) Default value of `ignore-open-flags` config parameter changed to `true`
+- [#1036](https://github.com/Azure/azure-storage-fuse/issues/1036) Panic if user does not have a $HOME directory set.
```

Comment:
The changelog entry here is kind of misleading - we should say something like "fix for panicking when $HOME dir is not set and respect `--default-working-dir` cli param setting"

Reply:
Updated